README.md

```diff
@@ -64,6 +65,12 @@ Finished in 542.64 milliseconds
 129 inspected, 2 failures
 ```
 
+### Watch a tutorial
+
+<a href="https://luckycasts.com/videos/ameba"><img src="https://i.imgur.com/uOETQlM.png" title="Write Better Crystal Code with the Ameba Shard" width="500" /></a>
```

Comment:
who controls the img on imgur? Can be deleted someday?

Reply:
I uploaded it on my account here:
https://imgur.com/user/stephendolan/posts

It can definitely be deleted (or at least set to "private"), though. Would having the file available on this PR be a good enough safety net for that case, or would you prefer to remove the image thumbnail altogether?

![ameba-crystal](https://user-images.githubusercontent.com/6677875/114557813-0b806e80-9c38-11eb-81df-0da9fe6e2e5b.png)

Reply:
Okay, thanks.